src/engine/ScreenFilter.tscn

```diff
@@ -0,0 +1,21 @@
+[gd_scene load_steps=5 format=2]
+
+[ext_resource path="res://src/engine/ScreenFilter.cs" type="Script" id=1]
+[ext_resource path="res://assets/textures/rusticyanin.png" type="Texture" id=2]
```

Comment:
This is just a random image, right? So I think that making this be `assets/misc/icon.png` would make a bit more sense than have the random organelle texture here, as those organelle textures are bigger in size and might be deleted in the future.